### ISSUE TITLE
Double windows

### ISSUE THREAD

**Issue by varlesh** [coslyk/moonplayer#89]
If open video file from the file manager (Dolphin on KDE also tested with Nautilus). Openned double windows - MPV & MoonPlayer:
![image](https://user-images.githubusercontent.com/8083855/79973298-61e52200-84a8-11ea-8382-e204ee803b38.png)
Also bug reproduced if run file from Terminal:
LOG:
```
/home/alex/Applications/MoonPlayer-3.2-alpha-x86_64_da2ae44a2e603d764ece981cb97f2ff7.appimage "/home/alex/Vidoe/file.mkv"
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
qrc:/qml/Explorer.qml:18:5: Unable to assign [undefined] to QObject*
[cplayer] Warning: option --hwdec-preload was replaced with --opengl-hwdec-interop and might be removed in the future.
[cplayer] Warning: option --opengl-hwdec-interop was replaced with --gpu-hwdec-interop and might be removed in the future.
[cplayer] Warning: option --video-aspect was replaced with --video-aspect-override and might be removed in the future.
[vo/gpu/opengl] Could not make context current!
```
Bug not reproduced if player already running and open video file from file manager or add from playlist
 
Sys info:
- KDE Neon (Ubuntu 18.04)
- MoonPlayer 3.2-alpha appimage

**Comment by coslyk** [comment]
Does it work with the stable version (v3.1) ?

**Comment by varlesh** [comment]
On 3.1 version program start with black screen and video file not played, but single window:
![image](https://user-images.githubusercontent.com/8083855/79978949-ceb0ea00-84b1-11ea-9af3-8e825ee3fe99.png)

LOG:
```
home/alex/Applications/MoonPlayer-3.1-x86_64_94793368bce3cbceb541cf85dec49f4c.appimage "/home/alex/Video.mkv"
Detected OpenGL backend: vdpau-glx
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
qrc:/qml/Explorer.qml:18:5: Unable to assign [undefined] to QObject*
MPV Error:  no audio or video data played
[vo/opengl-cb] OpenGL context not initialized.
[cplayer] Error opening/initializing the selected video_out (-vo) device.
```
if player already running - file openned from file manager and played normal with single window.

**Comment by varlesh** [comment]
Also strange if added `%U` option on start in Terminal - player open/play file normal on single window. But have error:
```
/home/alex/Applications/MoonPlayer-3.1-x86_64_94793368bce3cbceb541cf85dec49f4c.appimage %U "/home/alex/Видео/Video.mkv"
Detected OpenGL backend: vdpau-glx
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
libpng warning: iCCP: known incorrect sRGB profile
qrc:/qml/Explorer.qml:18:5: Unable to assign [undefined] to QObject*
MPV Error:  loading failed
[file] Cannot open file '%U': Нет такого файла или каталога
[stream] Failed to open %U.
```

**Comment by varlesh** [comment]
Maybe it's problem appimage integration?

**Comment by coslyk** [comment]
OMG according to the output it seems to be Nvidia's problem... Are you using Nvidia as your primary graphic card?

**Comment by varlesh** [comment]
Yes
```
alex@alex:~$ lspci -k | grep -EA3 'VGA|3D|Display'
01:00.0 VGA compatible controller: NVIDIA Corporation GP107 [GeForce GTX 1050 Ti] (rev a1)
        Subsystem: Gigabyte Technology Co., Ltd GP107 [GeForce GTX 1050 Ti]
        Kernel driver in use: nvidia
        Kernel modules: nvidiafb, nouveau, nvidia_drm, nvidia
```

**Comment by varlesh** [comment]
log from mpv 0.27.2:
```
mpv '/home/alex/Видео/Video.mkv'
Playing: Video.mkv
 (+) Video --vid=1 (*) (h264 1280x534 23.976fps)
     Video --vid=2 [P] 'cover.jpg' (mjpeg)
 (+) Audio --aid=1 --alang=rus (*) (f) (ac3 6ch 48000Hz)
 (+) Subs  --sid=1 --slang=rus (*) (f) 'Forced' (subrip)
     Subs  --sid=2 --slang=rus (subrip)
     Subs  --sid=3 --slang=ukr 'Forced' (subrip)
     Subs  --sid=4 --slang=ukr (subrip)
     Subs  --sid=5 --slang=eng 'Forced' (subrip)
     Subs  --sid=6 --slang=eng 'SDH' (subrip)
     Subs  --sid=7 --slang=eng (subrip)
File tags:
 Title: Release by Leonardo for Scarabey.org
VO does not support requested hardware decoder, or loading it failed.
AO: [pulse] 48000Hz 5.1(side) 6ch float
VO: [opengl] 1280x534 yuv420p
AV: 00:00:10 / 01:47:01 (0%) A-V:  0.000 Cache: 10s+73MB


Exiting... (Quit)
```

**Comment by varlesh** [comment]
mediainfo:
```
mediainfo *
General
Unique ID                                : 255942006327767147469852458003226986406 (0xC08CA32CBB9D5075E5CBD6C54B5C87A6)
Complete name                            : Video.mkv
Format                                   : Matroska
Format version                           : Version 2
File size                                : 3.03 GiB
Duration                                 : 1 h 47 min
Overall bit rate                         : 4 048 kb/s
Movie name                               : Release by Leonardo for Scarabey.org
Encoded date                             : UTC 2014-11-15 18:12:30
Writing application                      : mkvmerge v5.0.1 ('Es ist Sommer') built on Oct 23 2011 00:46:40
Writing library                          : libebml v1.2.2 + libmatroska v1.3.0
Cover                                    : Yes
Attachments                              : cover.jpg

Video
ID                                       : 1
Format                                   : AVC
Format/Info                              : Advanced Video Codec
Format profile                           : High@L4.1
Format settings                          : CABAC / 9 Ref Frames
Format settings, CABAC                   : Yes
Format settings, ReFrames                : 9 frames
Codec ID                                 : V_MPEG4/ISO/AVC
Duration                                 : 1 h 47 min
Bit rate                                 : 3 327 kb/s
Width                                    : 1 280 pixels
Height                                   : 534 pixels
Display aspect ratio                     : 2.40:1
Frame rate mode                          : Constant
Frame rate                               : 23.976 (24000/1001) FPS
Color space                              : YUV
Chroma subsampling                       : 4:2:0
Bit depth                                : 8 bits
Scan type                                : Progressive
Bits/(Pixel*Frame)                       : 0.203
Stream size                              : 2.49 GiB (82%)
Writing library                          : x264 core 125 r2208 d9d2288
Encoding settings                        : cabac=1 / ref=9 / deblock=1:-3:-3 / analyse=0x3:0x133 / me=umh / subme=11 / psy=1 / psy_rd=1.00:0.00 / mixed_ref=1 / me_range=32 / chroma_me=1 / trellis=2 / 8x8dct=1 / cqm=0 / deadzone=21,11 / fast_pskip=0 / chroma_qp_offset=-2 / threads=12 / lookahead_threads=2 / sliced_threads=0 / nr=0 / decimate=1 / interlaced=0 / bluray_compat=0 / constrained_intra=0 / bframes=8 / b_pyramid=2 / b_adapt=2 / b_bias=0 / direct=1 / weightb=1 / open_gop=0 / weightp=2 / keyint=250 / keyint_min=23 / scenecut=40 / intra_refresh=0 / rc=crf / mbtree=0 / crf=18.0 / qcomp=0.60 / qpmin=0 / qpmax=69 / qpstep=4 / ip_ratio=1.10 / pb_ratio=1.10 / aq=1:1.00
Language                                 : English
Default                                  : Yes
Forced                                   : No

Audio
ID                                       : 2
Format                                   : AC-3
Format/Info                              : Audio Coding 3
Codec ID                                 : A_AC3
Duration                                 : 1 h 47 min
Bit rate mode                            : Constant
Bit rate                                 : 640 kb/s
Channel(s)                               : 6 channels
Channel positions                        : Front: L C R, Side: L R, LFE
Sampling rate                            : 48.0 kHz
Frame rate                               : 31.250 FPS (1536 SPF)
Bit depth                                : 16 bits
Compression mode                         : Lossy
Stream size                              : 490 MiB (16%)
Language                                 : Russian
Service kind                             : Complete Main
Default                                  : Yes
Forced                                   : Yes

Text #1
ID                                       : 3
Format                                   : UTF-8
Codec ID                                 : S_TEXT/UTF8
Codec ID/Info                            : UTF-8 Plain Text
Title                                    : Forced
Language                                 : Russian
Default                                  : Yes
Forced                                   : Yes

Text #2
ID                                       : 4
Format                                   : UTF-8
Codec ID                                 : S_TEXT/UTF8
Codec ID/Info                            : UTF-8 Plain Text
Language                                 : Russian
Default                                  : No
Forced                                   : No

Text #3
ID                                       : 5
Format                                   : UTF-8
Codec ID                                 : S_TEXT/UTF8
Codec ID/Info                            : UTF-8 Plain Text
Title                                    : Forced
Language                                 : Ukrainian
Default                                  : No
Forced                                   : No

Text #4
ID                                       : 6
Format                                   : UTF-8
Codec ID                                 : S_TEXT/UTF8
Codec ID/Info                            : UTF-8 Plain Text
Language                                 : Ukrainian
Default                                  : No
Forced                                   : No

Text #5
ID                                       : 7
Format                                   : UTF-8
Codec ID                                 : S_TEXT/UTF8
Codec ID/Info                            : UTF-8 Plain Text
Title                                    : Forced
Language                                 : English
Default                                  : No
Forced                                   : No

Text #6
ID                                       : 8
Format                                   : UTF-8
Codec ID                                 : S_TEXT/UTF8
Codec ID/Info                            : UTF-8 Plain Text
Title                                    : SDH
Language                                 : English
Default                                  : No
Forced                                   : No

Text #7
ID                                       : 9
Format                                   : UTF-8
Codec ID                                 : S_TEXT/UTF8
Codec ID/Info                            : UTF-8 Plain Text
Language                                 : English
Default                                  : No
Forced                                   : No

Menu
00:00:00.000                             : en:00:00:00.000
00:01:29.172                             : en:00:01:29.172
00:04:47.579                             : en:00:04:47.579
00:07:43.672                             : en:00:07:43.672
00:12:07.894                             : en:00:12:07.894
00:17:48.943                             : en:00:17:48.943
00:20:01.325                             : en:00:20:01.325
00:22:51.537                             : en:00:22:51.537
00:26:34.634                             : en:00:26:34.634
00:30:30.161                             : en:00:30:30.161
00:32:41.960                             : en:00:32:41.960
00:34:13.509                             : en:00:34:13.509
00:37:15.566                             : en:00:37:15.566
00:40:44.150                             : en:00:40:44.150
00:42:53.154                             : en:00:42:53.154
00:46:18.567                             : en:00:46:18.567
00:46:57.940                             : en:00:46:57.940
00:49:48.527                             : en:00:49:48.527
00:53:56.399                             : en:00:53:56.399
00:54:57.252                             : en:00:54:57.252
00:57:36.202                             : en:00:57:36.202
00:59:27.730                             : en:00:59:27.730
01:02:16.941                             : en:01:02:16.941
01:05:56.285                             : en:01:05:56.285
01:12:31.764                             : en:01:12:31.764
01:14:05.899                             : en:01:14:05.899
01:15:24.186                             : en:01:15:24.186
01:21:12.992                             : en:01:21:12.992
01:23:22.247                             : en:01:23:22.247
01:29:06.924                             : en:01:29:06.924
01:31:48.127                             : en:01:31:48.127
01:32:41.222                             : en:01:32:41.222
01:38:18.600                             : en:01:38:18.600
```

**Comment by coslyk** [comment]
Maybe you can try the Flatpak version, the AppImage doesn't work well with graphic drivers

**Comment by coslyk** [comment]
The problem should be fixed in the [v3.3-alpha](https://github.com/coslyk/moonplayer/releases/tag/v3.3-alpha) version.
